utilities/rali/rali_unittests/rali_unittests.cpp

```diff
@@ -157,7 +158,17 @@ int test(int test_case, const char *path, const char *outName, int rgb, int gpu,
         std::cout << "\n json_path has to be set in rali_unit test manually";
         exit(0);
     }
+
     meta_data = raliCreateCOCOReader(handle, json_path, true);
+#elif defined COCO_READER_KEYPOINTS
```

Comment:
Please give a command_line option for each reader type instead of relying on #defines

Reply:
We are creating a separate PR for this as we discussed